export.py

```diff
@@ -1,3 +1,22 @@
+"""FRDR Harvester
+
+Usage:
+  harvest.py [--openrefine-import | --onlyharvest | --onlyexport | --init] [--only-new-records] [--dump-on-failure] [--export-filepath=<file>] [--export-format=<format>] [--repository-id=<id>] [--openrefine-csv=<file>]
+
+Options:
+  --openrefine-import       Don't harvest or export normally; import data from OpenRefine.
+  --onlyharvest             Just harvest new items, do not export anything.
+  --onlyexport              Just export existing items, do not harvest anything.
+  --only-new-records        Only export records changed since last crawl.
+  --dump-on-failure         If a record ever fails validation, print the whole record.
+  --export-filepath=<file>  The path to export the data to.
+  --openrefine-csv=<file>   The CSV from OpenRefine to import.
+  --export-format=<format>  The export format (currently gmeta or dataverse).
+  --repository-id=<id>      Only export this repository, based on the database table ID
+  --init                    Just initialize the database, do not harvest or export.
+
+"""
+
```

Comment:
If there isn't something for the program to pull into __doc__ there is a failure in the docopt.py when it tries to run `usage_split = re.split(r'([Uu][Ss][Aa][Gg][Ee]:)', doc)`, and the whole program crashes. I didn't need to pull the full docopt section from the harvest.py, but figured it didn't hurt to have since export.py is really just the harvest.py with the export flags.

Reply:
Ah that's probably a better question to ask then: why does this script exist? Can't one just run harvest.py --onlyexport ?

Reply:
Now that you mention it, I think export.py was not supposed to get saved to github.